docetl/utils.py

```
@@ -17,23 +18,18 @@ def __init__(self, secret_key: str):
         self.lz = LZString()
 
     def decrypt(self, encrypted_data: str) -> str:
-        try:
-            # First decompress the data
-            compressed = self.lz.decompressFromBase64(encrypted_data)
-            if not compressed:
-                raise ValueError("Invalid compressed data")
+        # First decompress the data
+        compressed = self.lz.decompressFromBase64(encrypted_data)
+        if not compressed:
+            raise ValueError("Invalid compressed data")
 
-            # Then decode using the key
-            result = ""
-            for i in range(len(compressed)):
-                char_code = ord(compressed[i]) - ord(self.key[i % len(self.key)])
-                result += chr(char_code)
+        # Then decode using the key
+        result = ""
+        for i in range(len(compressed)):
+            char_code = ord(compressed[i]) - ord(self.key[i % len(self.key)])
+            result += chr(char_code)
 
-            return result
-
-        except Exception as e:
-            print(f"Decryption failed: {str(e)}")
-            return None
```

Changed it so that the exceptions bubble up to the caller. The alternative would be to modify the `decrypt` signature from `-> str` to be `-> str | None`, but I think failing fast makes sense.

---

Makes sense. this is only used for DocWrangler